src/connections/ble_handler.rs

```diff
@@ -41,10 +41,12 @@ pub enum RadioMessage {
 
 /// Bluetooth Low Energy ID, used to filter available devices.
 pub enum BleId {
```

Comment:
In my code I wanted to derive Debug and CLone for this struct:
```
#[derive(Debug, Clone)]
pub enum DiscoveryEvent {
    BLERadioFound(BleId),
    BLERadioLost(BleId),
}
```

but neither are provided for BleId

Reply:
Added